src/components/refactor/dialogs/delete-voltage-level-on-line/delete-voltage-level-on-line-dialog.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { useSnackMessage } from '@gridsuite/commons-ui';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+    LINE_TO_ATTACH_TO_1_ID,
+    LINE_TO_ATTACH_TO_2_ID,
+    REPLACING_LINE_ID,
+    REPLACING_LINE_NAME,
+} from 'components/refactor/utils/field-constants';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { deleteVoltageLevelOnLine } from '../../../../utils/rest-api';
+import { sanitizeString } from '../../../dialogs/dialogUtils';
+import yup from '../../utils/yup-config';
+import ModificationDialog from '../commons/modificationDialog';
+import DeleteVoltageLevelOnLineForm from './delete-voltage-level-on-line-form';
+
+const emptyFormData = {
+    [LINE_TO_ATTACH_TO_1_ID]: '',
+    [LINE_TO_ATTACH_TO_2_ID]: '',
```

Comment:
use null default value when dealing with selectInput / Autocomplete
make those fields nullable in yup as well

currently, if you clear the field, you will get a wrong error message

Reply:
yes, I currently get "bad format" with an empty field.
with null / nullable, I got "this field must be set".